Python/pystate.c

```diff
@@ -556,6 +556,11 @@ init_interpreter(PyInterpreterState *interp,
 #ifdef Py_GIL_DISABLED
     _Py_brc_init_state(interp);
 #endif
+
+#ifdef _Py_TIER2
+     // Ensure the buffer is to be set as NULL.
+    // interp->jit_uop_buffer = NULL;
```

Comment:
I think the bug is here? You didn't set it to NULL. Remember that MSVC doesn't necessarily initialize values.

Reply:
No, I just tested for all scenarios that I can expect.